modules/insights-operator-new-pull-secret-enable.adoc

```diff
@@ -15,7 +15,8 @@ You can modify your existing global cluster pull secret to enable remote health
 
 .Procedure
 
-. Navigate to link:https://console.redhat.com/openshift/downloads[https://console.redhat.com/openshift/downloads].
+. Go to the link:https://console.redhat.com/openshift/downloads[Downloads] page on the {hybrid-console}.
+
 . From *Tokens* -> *Pull Secret*, click *Download*.
```

Comment:
see from <img width="1902" height="647" alt="Image" src="https://github.com/user-attachments/assets/c397ccbd-3009-410a-9963-113a7a84635e" />
it is `Pull secret`